src/components/PopupWithForm.js

```diff
@@ -14,26 +14,36 @@ class PopupWithForm extends Popup {
     inputs.forEach((input) => {
       values[input.name] = input.value;
     });
+
     return values;
   }
 
   _submitForm = () => {
     const inputValues = this._getInputValues();
+    this._loadingState = true;
+    this.showLoading(true);
     this._handleFormSubmit(inputValues);
   };
 
+  showLoading(isLoading) {
```

Comment:
Needs correction: Please add the default value here, so when you call `showLoading` without arguments the loading state will be shown ` showLoading(isLoading = true) {`